### PR TITLE
Docs: adds TCP reference for pomerium-cli

### PR DESCRIPTION
**Pull request by ZPain8464**

### CODE REVIEW

File: k8s/console/deployment/args.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ spec:
     spec:
       containers:
         - name: pomerium-console
+          args: ['serve']
```
Comment (desimone) [inline k8s/console/deployment/args.yaml:10]: Seems unrelated?

File: content/docs/capabilities/tcp/reference/tcp-cli-reference.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Flags
+
+| Flags | Description | Default Value |
+| :-- | :-- | --- |
+| <a className="entRef-anchor" id="--alternate-ca-path">#</a><a href='#--alternate-ca-path'>--alternate-ca-path</a> | Path to CA certificate to use for HTTP requests. | string |
+| <a className="entRef-anchor" id="--browser-cmd">#</a><a href='#--browser-cmd'>--browser-cmd</a> | Custom browser command to run when opening a URL. | string |
+| <a className="entRef-anchor" id="--ca-cert">#</a><a href='#--ca-cert'>--ca-cert</a> | Path to CA certificate to use for HTTP requests. | string |
+| <a className="entRef-anchor" id="--client-cert">#</a><a href='#--client-cert'>--client-cert</a> | (optional) PEM-encoded client certificate. | string |
+| <a className="entRef-anchor" id=" --client-key">#</a><a href='# --client-key'> --client-key</a> | (optional) PEM-encoded client certificate. | string |
```
Comment (desimone) [inline content/docs/capabilities/tcp/reference/tcp-cli-reference.md:27]:
```suggestion
| <a className="entRef-anchor" id=" --client-key">#</a><a href='# --client-key'> --client-key</a> | (optional) PEM-encoded client certificate key. | string |
```

**Review by desimone — CHANGES_REQUESTED** [review]
LGTM except there seems to be an unrelated change and an included empty file.

File: k8s/console/deployment/args.yaml (hunk 1 above)
Comment (ZPain8464) [reply k8s/console/deployment/args.yaml:10]: Sorry, some of these unrelated changes are a result of running the pre-commits. Specifically, cspell seems to always modify k8s files. I may need to talk to Denis to figure this out so it doesn't continue on future PRs.

Comment (calebdoxsey) [reply k8s/console/deployment/args.yaml:11]: https://github.com/pomerium/documentation/pull/303

Comment (ZPain8464) [reply k8s/console/deployment/args.yaml:11]: @calebdoxsey didn't realize that was what was needed, will keep in mind going forward. Thanks!

**Review by cmo-pomerium — APPROVED** [review]

File: k8s/console/deployment/args.yaml [hunk 3]
```diff
@@ -7,6 +7,11 @@ spec:
     spec:
       containers:
         - name: pomerium-console
+<<<<<<< HEAD
```
Comment (desimone) [inline k8s/console/deployment/args.yaml:10]: Merge conflicts still here.

File: k8s/console/deployment/args.yaml [hunk 4]
```diff
@@ -7,6 +7,11 @@ spec:
...
+          args: ['serve']
           args:
             - --tls-derive=pomerium-console.$(POMERIUM_ENTERPRISE_NAMESPACE).svc.cluster.local
             - serve
+=======
```
Comment (desimone) [inline k8s/console/deployment/args.yaml:15]: And here.

**Review by desimone — CHANGES_REQUESTED** [review]
Still seeing merge conflicts.

**Review by desimone — APPROVED** [review]
